_sass/content.scss

```diff
@@ -82,21 +82,15 @@
     }
   }
 
-  .task-list {
-    padding-left: 0;
-  }
-
   .task-list-item {
-    display: flex;
-    align-items: center;
-
     &::before {
       content: "";
     }
   }
 
   .task-list-item-checkbox {
     margin-right: 0.6em;
+    margin-left: -1.4em;
```

Comment:
Could I ask - is there a reason why we've chosen this specific number?

Reply:
The same `margin-left` is used above for `ul > li::before`. When using an inspector in the browser to check the margins, it seemed to me that the checkbox in a `task-list-item` should be positioned in the same way. The added tests looked okay to me.

Reply:
Understood - do you mind adding that as a quick comment somewhere? I'm also concerned if we change `ul > li::before` later down the line.